code/modules/cm_marines/equipment/sentries.dm

```diff
@@ -1141,33 +1140,39 @@
 */
 /obj/machinery/marine_turret/premade
 	name = "UA-577 Gauss Turret"
+	desc = "A deployable, semi-automated turret with AI targeting capabilities. Armed with an armor penetrating MIC Gauss Cannon and a high-capacity drum magazine."
+	ammo = /datum/ammo/bullet/turret/gauss //This is a gauss cannon; it will be significantly deadlier
 	immobile = TRUE
 	on = TRUE
 	burst_fire = TRUE
-	rounds = 50000
 	rounds_max = 50000
 	icon_state = "sentry_base"
 
-/obj/machinery/marine_turret/premade/New()
-	spark_system = new /datum/effect_system/spark_spread
-	spark_system.set_up(5, 0, src)
-	spark_system.attach(src)
+/obj/machinery/marine_turret/premade/Initialize()
```

Comment:
If you change it to `Initialize()` and there's a `	. = ..()` you need to go down that well and change the `New()` that get called there, and also go up and change the `New()` procs that may have been calling this one.

Reply:
I believe this has been handled

Reply:
Are you sure all the children and parents of this proc are now `Initialize()`?